pkg/otlp/metrics/statspayload_test.go

```diff
@@ -487,6 +487,32 @@ func TestConversion(t *testing.T) {
 		if found != len(want.Stats) {
 			t.Fatalf("Found %d/%d", found, len(want.Stats))
 		}
+		mx, err := trans.StatsToMetrics(want)
+		assert.NoError(t, err)
+		var results []*pb.StatsPayload
+		for i := 0; i < mx.ResourceMetrics().Len(); i++ {
+			rm := mx.ResourceMetrics().At(i)
+			for i := 0; i < rm.ScopeMetrics().Len(); i++ {
```

Comment:
nit: `i` is overridden
```suggestion
			for j := 0; j < rm.ScopeMetrics().Len(); j++ {
```